packages/dropdowns/src/views/menu/StyledButton.ts

```diff
@@ -1,23 +0,0 @@
-/**
- * Copyright Zendesk, Inc.
- *
- * Use of this source code is governed under the Apache License, Version 2.0
- * found at http://www.apache.org/licenses/LICENSE-2.0.
- */
-
-import styled from 'styled-components';
-import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
-import { Button } from '@zendeskgarden/react-buttons';
-
-const COMPONENT_ID = 'dropdowns.menu.button';
-
-export const StyledButton = styled(Button).attrs({
-  'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION
-})`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
-`;
```

Comment:
RE: https://github.com/zendeskgarden/react-components/pull/1804#discussion_r1603476175

After removing all code causing a breaking-change, we would be left with:

```
export const StyledButton = styled(Button)``
```

So I think it's better to delete.